run-bench.sh

```diff
@@ -5,13 +5,33 @@ Usage: run-bench.sh [OPTIONS] SIMULATOR
 Run NSuite benchmarks for SIMULATOR.
 
 Options:
-    --prefix=PATH    Use PATH as base for working directories.
-    --model=MODEL    Run benchmark MODEL.
-    --config=CONFIG  Run benchmarks with configuration CONFIG.
-    SIMULATOR        One of: arbor, neuron, or coreneuron.
+    --prefix=PATH        Use PATH as base for working directories.
+    --model=MODEL        Run benchmark MODEL.
+    --config=CONFIG      Run benchmarks with configuration CONFIG.
+    -o, --output=FORMAT  Override default path to benchmark outputs.
+    SIMULATOR            One of: arbor, neuron, or coreneuron.
 
 --model and --config can be supplied multiple times. If omitted, the ring
 benchmark will be run with the small configuration.
+
```

Comment:
I would consider short/long help, or at least `type --help format for more info`